manifests/management.pp

```diff
@@ -1,14 +1,16 @@
 # Management Server.
+class rancher::management(
+  $rancher_manager_port = '8080',
+){
 
-class rancher::management {
+  require ::docker
```

Comment:
Using require explictly makes sure that the docker class has been applied before all the resources in the class are. Using include does not, details here: https://docs.puppetlabs.com/puppet/latest/reference/lang_classes.html#using-require